geo.js

```diff
@@ -69,7 +74,14 @@ function fit(bounds, width, height)
   return scale;
 }
 
-// D3.js projection from GeoJSON bounds
+/**
+* D3.js projection from GeoJSON bounds
+* @param {*} geojson
```

Comment:
Can you also explain what the parameters are? Would be hard for most people to understand what to pass in otherwise. feel free to ask me what the params are.